builtin/common/misc_helpers.lua

```diff
@@ -517,6 +517,63 @@ function table.shuffle(t, from, to, random)
 end
 
 
+function table.find(t, e, i, j)
+    i = i or 1
+    j = j or #t
+    
+    local mt = {__eq = function(e1, e2)
+            if type(e2) ~= "table" then
+                return false
+            end
+             
+            local is_elem_exists = true
+            for i1, v1 in pairs(e1) do
+                if v1 ~= e2[i1] then
+                    is_elem_exists = false
+                end  
+            end
+                    
+            return is_elem_exists
+    end}
+  
+    if type(e) == "table" then
+        setmetatable(table.copy(e), mt)
```

Comment:
This needs to be assigned to a variable because while you create the copy and set the metatable, this is a new table and is unrelated to the existing one.

Just add `e = ` to the start of this line.

Reply:
Done.